src/modals/Transfer.tsx

```diff
@@ -128,7 +128,8 @@ function Transfer(props: TransferProps) {
             };
 
             const txDetails = await state.provider!.relayTransaction(
-                relayTrxOpts
+                relayTrxOpts,
+                true
```

Comment:
In order to use this parameter in this way, we may need to change the SDK [interface](https://github.com/rsksmart/rif-relay-sdk/blob/master/src/index.ts#L70) and [method](https://github.com/rsksmart/rif-relay-sdk/blob/master/src/sdk.ts#L302). Maybe we could consider just adding an optional parameter in the [RelayTransactionOptions](https://github.com/rsksmart/rif-relay-sdk/blob/master/src/interfaces.ts#L52).
Furthermore, have you considered supporting the exact mechanism for the deployment?

Reply:
In the deployment, the logic for waiting for the receipt is on the dapp. When you refer to this, you mean to move the logic to the rif-relay-client so they can have the option to decide to wait but from the API itself

Reply:
After discussing, we managed to include the same logic for the deployment as well. Thank you for the changes!